### DISCUSSION TITLE
Sending Sounds Different Sounds to Different Controllers

### DISCUSSION THREAD

**Discussion by garamoniker** [GeekFunkLabs/fluidpatcher#70] [General]
Hey all,

Played my first gig with the Squishbox setup and it absolutely killed. After years of using Ableton and Mainstage, I can't tell you how good it felt to just connect my controller, plug in a jack cable and start playing.

I'm wondering if I can now send different sf2s to different controllers?

Some thing like this:

Controller 1: (Midi Channel 1): Trumpet.sf2
Controller 2: (Midi Channel 2): Keys.sf2 

Just to clarify, both instruments will be playing around the same range (different parts) - hence why I can't just set up the sounds on different octaves, e.t.c.

Thanks, guys!

**Comment by garamoniker** [comment]
Ha! I'm an idiot. Been fooling around with split voices so much I forgot:

The fist number in the patch parameter is the midi channel.

**Comment by albedozero** [comment]
Awesome! If you ever feel like sharing your patches/bank files as an example or related :wink: pics/videos of performance/setup feel free.
